EDI/UserInterface/00_UserInterface.md

```diff
@@ -0,0 +1,31 @@
+# User Interface
+
+In this part of the *EDIFACT* documentation, you will find the following chapters:
+
+- [Exports](./01a_Exports.md)
+    - [Create exporter definition](./01a_Exports.md#create-exporter-definition)
+    - [Edit exporter definition](./01a_Exports.md#edit-exporter-definition)
+    - [Settings tab](./01a_Exports.md#settings-tab)
+    - [File settings tab](./01a_Exports.md#file-settings-tab)
```

Comment:
In der Regel schreiben wir hier nicht "tab", sondern nur den Namen der Tabs: Settings, File settings, Data sources...
Prinzipiell sollten die vier Tabs dem Edit exporter definition UI-Datei untergeordnet sein, also auch eingerückt. Step1-4 (wizard window) wären dann nicht mehr im TOC angezeigt (da H4). So die Regeln. Macht es Sinn hier? Brauchen wir alle Steps des Wizard-Fensters hier aufzulisten?

Reply:
done, fände ich schon sinnvoll, es im TOC zu haben, außer dem ist es ja nicht nur Edit, sondern auch Create